backend/docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+version: '3'
+services:
+  sqlite-server:
```

Comment:
We don't need this service to be separated because sqlLite is not real database that runs as separate service. It is integrated to the main app.